entry_types/scrolled/package/src/editor/vendor/jquery.imgareaselect.js

```diff
@@ -282,6 +284,15 @@ $.imgAreaSelect = function (img, options) {
         maxWidth = round(min(options.maxWidth / scaleX || 1<<24, imgWidth));
         maxHeight = round(min(options.maxHeight / scaleY || 1<<24, imgHeight));
 
+        if (imgWidth > imgHeight) {
+            gridX = imgWidth / gridSteps;
+            gridY = gridX + (imgHeight % gridX) / Math.floor(imgHeight / gridX);
```

Comment:
An edge case indeed, but still something I missed. I wasn't even able, though, to upload a file a like that since apparently something in our Paperclip processing does not like such extreme image dimensions. Still, it's easy enough to handle the case here. Updated.